resource/src/models/index.mustache

```diff
@@ -0,0 +1,9 @@
+const util = require('../utils/module');
+const models = util.slurp('src/models').obj;
+
+// making all the models to be exported pacal cased
+Object.keys(models).forEach(key => {
+  const pascalCasedName = key[0].toUpperCase() + key.substr(1);
+
+  module.exports[pascalCasedName] = models[key];
+});
```

Comment:
What's the purpose of exporting these as a dict from an index file? Why not just `require` them directly?